libs/components/src/next/infotext/vl-infotext.css.ts

```diff
@@ -22,7 +22,7 @@ const styles: CSSResult = css`
 
         &.vl-infotext--badge {
             background-color: var(--vl-background-alt-color);
-            border: 1px solid var(--vl-border-color);
+            border: 1px solid var(--vl-alt-border-color);
```

Comment:
Vermoed dat dit bewust is maar kan je deze wijziging even toelichten?

Reply:
Bij nader inzien lijkt deze border-color minder vaak voor te komen, vandaar de alt-border benaming.